sheepdog/transactions/upload/sub_entities.py

```diff
@@ -368,29 +435,32 @@ def _is_valid_index_for_file(self):
 
     def _is_valid_index_id_for_graph(self):
         # Do not check by hash and size if uniqueness is not enforced (for ex. GDC)
-        if not flask.current_app.config.get('ENFORCE_FILE_HASH_SIZE_UNIQUENESS', True):
+
```

Comment:
Is that right? Plz add `CREATE_REPLACEABLE` flag to `dev_settings` along with explanation what it does